test/behaviour/connection/session/SessionSteps.java

```diff
@@ -40,27 +43,61 @@
 
 public class SessionSteps {
 
-    @When("connection open session for keyspace: {word}")
-    public void connection_open_session_for_keyspace(String name) {
-        connection_open_sessions_for_keyspaces(list(name));
+    @When("connection open schema session for database: {word}")
+    public void connection_open_schema_session_for_database(String name) {
+        connection_open_schema_sessions_for_databases(list(name));
     }
 
-    @When("connection open session(s) for keyspace(s):")
-    public void connection_open_sessions_for_keyspaces(List<String> names) {
+    @When("connection open (data )session for database: {word}")
+    public void connection_open_data_session_for_database(String name) {
+        connection_open_data_sessions_for_databases(list(name));
+    }
+
+    @When("connection open schema session(s) for database(s):")
+    public void connection_open_schema_sessions_for_databases(List<String> names) {
+        for (String name : names) {
+            sessions.add(client.schemaSession(name));
+        }
+    }
+
+    @When("connection open (data )session(s) for database(s):")
+    public void connection_open_data_sessions_for_databases(List<String> names) {
         for (String name : names) {
             sessions.add(client.session(name));
         }
     }
 
-    @When("connection open sessions in parallel for keyspaces:")
-    public void connection_open_sessions_in_parallel_for_keyspaces(List<String> names) {
+    @When("connection open (data )sessions in parallel for databases:")
```

Comment:
will this work if there's a mismatch in the spacing in the BDD scenario definition?

Reply:
no

Reply:
So where is the typo - here `(data )` ? or In verification do we need to change it to be correct `(data)`?